### ISSUE TITLE
Proxy not working

### ISSUE THREAD

**Issue by SrMilton** [GermanAizek/webdriverxx#14]
Sup dude, me again.

I can't use proxy in chrome.

Code:
>```c++
>WebDriver gc = Start(Chrome().SetProxy(
>        SocksProxy("186.179.7.98:8175")
>    ));
>```

Result:

![image](https://user-images.githubusercontent.com/35127424/114063950-867c0a80-986f-11eb-96fb-0874522bed6b.png)

**Comment by GermanAizek** [comment]
![](https://cdn.discordapp.com/attachments/735477376367984641/743097116989063278/647625114170687488.png)

**Comment by GermanAizek** [comment]
@SrMilton, Oh shit, I'm sorry.

But it surprises me that the int type is already available.

https://github.com/GermanAizek/webdriverxx/blob/ce7954bd1f354605ff4cff63d3742c77e9315624/src/include/webdriverxx/capabilities.h#L170

Amazing what kind of int requires chromedriver XD

**Comment by GermanAizek** [comment]
@SrMilton,
I noticed you didn't specify a sockы version. For simplicity I have to make socks by default 5, thanks for the advice.

**Comment by GermanAizek** [comment]
@SrMilton,

manual set proxy version
```cpp
WebDriver gc = Start(Chrome().SetProxy(
       SocksProxy("186.179.7.98:8175").SetProxyVersion(4);
   ));
```

now in the new commit you can do this https://github.com/GermanAizek/webdriverxx/commit/5156a4b1d901652a676ab368770bcdc7edf4f047:

default set proxy version 5
```cpp
WebDriver gc = Start(Chrome().SetProxy(
       SocksProxy("186.179.7.98:8175");
   ));
```

**Comment by SrMilton** [comment]
When i use 
>```c++
>SocksProxy("186.179.7.98:8175").SetProxyVersion(4);
>```
it crash and when i don't use SetProxyVersion, it just ignore the proxy and open with my normal ip.

![image](https://user-images.githubusercontent.com/35127424/114087059-c7811880-9889-11eb-85c6-899829e8cee9.png)

**Comment by GermanAizek** [comment]
maybe this bug
https://github.com/GermanAizek/webdriverxx/issues/13#issue-843991800

I need to understand why chromedriver does not see api json files.

**Comment by SrMilton** [comment]
So this should work in firefox?

**Comment by GermanAizek** [comment]
> So this should work in firefox?

@SrMilton ,
I found a solution to the problem not only with the socks version.
Everything turned out to be very simple, the developers still have an old release version. Conversion bug int to long, a year ago has been fixed, but is in beta version selenium server.

More info here: https://github.com/mablhq/selenium/commit/801be37639e0050d976dc2972fc236da334f175a

The conclusion is as follows, do not use the release version yet.
Api works perfectly with > 4.x selenium server.

I will explain in more detail in the README.md how to configure and run a new beta server. Selenium developers are not ready to update the version of the release, but it has much more features and fewer bugs.
